src/dymaptic.Blazor.GIS.API.Core/wwwroot/arcGisJsInterop.js

```diff
@@ -1004,4 +1005,24 @@ export function getActiveWidgetsForView(viewId) {
     let realWidgets = arcGisObjectRefs[viewId]?.ui?._components.filter(c => c?.widget !== undefined).map(c => c.widget);
     let registeredWidgets = Object.values(arcGisObjectRefs).filter(o => o?.declaredClass.includes('esri.widgets'));
     return realWidgets?.filter(wc => registeredWidgets.find(r => r === wc));
+}
+
+
+function waitForRender(viewId) {
```

Comment:
Are all of these (and the updates for Firefox) in the TypeScript branch?

Reply:
No, they are all separate branches/prs.